examples/pokemon-service-client-usage/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "pokemon-service-client-usage"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[features]
+
+
+[dependencies]
+aws-smithy-async = { path = "../../rust-runtime/aws-smithy-async/", features = ["rt-tokio"] }
+aws-smithy-client = { path = "../../rust-runtime/aws-smithy-client/" }
+aws-smithy-http = { path = "../../rust-runtime/aws-smithy-http/" }
+aws-smithy-types = { path = "../../rust-runtime/aws-smithy-types/" }
+aws-smithy-runtime-api = { path = "../../rust-runtime/aws-smithy-runtime-api/" }
+aws-smithy-runtime = { path = "../../rust-runtime/aws-smithy-runtime/", features=["test-util"] }
```

Comment:
As much as possible, you should try to avoid referencing the runtime crates directly. Most types are re-exported in the generated client crate. The exception to that right now is the HyperClientBuilder, but we should fix that at some point...

Reply:
Makes sense. I've now only kept the ones that the examples need and are not currently being re-exported.